<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Cambria</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Login</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>480</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="parent" native="true">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="signin" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(188, 188, 188);
border-radius: 9px;
</string>
         </property>
         <widget class="QPushButton" name="signin_btn">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>380</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Cambria</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 13px;</string>
          </property>
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
         <widget class="QPushButton" name="register_btn">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>380</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Cambria</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 13px;</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="signup" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 9px;</string>
         </property>
         <widget class="QPushButton" name="signup_btn">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>380</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Cambria</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(188, 188, 188);
border-radius: 13px;</string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
         <widget class="QPushButton" name="cancel_register_btn">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>380</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Cambria</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(188, 188, 188);
border-radius: 13px;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
